TheRocket/TheRocket.ipynb

```diff
@@ -10,6 +10,209 @@
     "- Adhere to units seen in comments [conversion done in 'Solver Run']"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read in data from spreadsheet at ./data/variables.xlsx\n",
+    "import pandas as pd\n",
+    "def readVars(sheet, skiprows, nrows):\n",
+    "    data = pd.read_excel('./data/variables.xlsx', sheet_name=sheet, skiprows = skiprows, nrows = nrows)\n",
+    "    # Variable \tValue\tUnit\tDescription\tFinalized\n",
+    "    data.columns = ['Variable', 'Value', 'Unit', 'Description', 'Finalized']\n",
+    "    print(data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "                    Variable  Value Unit  Description  Finalized\n",
+      "0           vehicle_mass_dry     79   lb          NaN        NaN\n",
```

Comment:
It looks like the code is always displaying NaN for the finalized column. Would it be worthwhile to explicitly set it to False or something in the `readVars` function?

Reply:
I fixed this so now it sets that value to the variable and doesn't output the entire table. In the excel sheet the finalized column contains a check box which Aaron had requested, but the only columns the code requires are 'Variable' and 'Value'

Reply:
Sounds good!